### PR TITLE
PPF-186 - Admin can now activate individual auth0 clients

### PR DESCRIPTION
**Pull request by grubolsch**
### Added
Admin can now activate individual auth0 clients

---
Ticket: https://jira.uitdatabank.be/browse/PPF-186

### CODE REVIEW

**Review by LucWollants — CHANGES_REQUESTED** [review]
Same remark as for the block action. There is a difference between an `Event`, something that happened and was persisted, and a `Command` or `Job` which is the intention to modify.

**Comment by grubolsch** [comment]
> Same remark as for the block action. There is a difference between an `Event`, something that happened and was persisted, and a `Command` or `Job` which is the intention to modify.

I have updated the PR with your feedback.

File: app/Auth0/Jobs/ClientActivatedListener.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            $this->logger->error(
+                'Failed to block Auth0 client',
+                [
+                    'domain' => 'auth0',
+                    'message' => $e->getMessage(),
+                    'id' => $event->id,
+                ]
+            );
+        }
+        $this->logger->info(
```
Comment (LucWollants) [inline app/Auth0/Jobs/ClientActivatedListener.php:38]: I think this success message is traced even when the model was not found.

File: app/Auth0/Jobs/ClientActivatedListener.php [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+namespace App\Auth0\Jobs;
+
+use App\Auth0\Auth0ClusterSDK;
+use App\Auth0\Events\ClientActivated;
+use App\Auth0\Repositories\Auth0ClientRepository;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Psr\Log\LoggerInterface;
+
+final class ClientActivatedListener implements ShouldQueue
```
Comment (LucWollants) [inline app/Auth0/Jobs/ClientActivatedListener.php:14]: The name should be `ActivateClientListener`

**Pull request merged**